deployer/deployer.py

```diff
@@ -527,8 +540,6 @@ def deploy(self, environment='pre', eureka=True, health_check=None, health_check
             self.set_scheduler(version)
 
     def deploy_elb(self, health_check, health_check_port):
-        self.elbs = [self.app]
-
```

Comment:
I've changed this fix. It doesn't work for me. I changed 'self.app' by 'self.get_cluster_name()', and it works now for me.